ci-templates/gitlab/image-builder.yml

```diff
@@ -96,6 +108,28 @@ default:
   - cd $CAPELLA_VERSION
   - *github
 
+.local-git-server: &local-git-server
+  - |
+    docker build $DOCKER_BUILD_ARGS \
+      -t $T4C_SERVER_REGISTRY/local-git-server \
+      --build-arg BASE_IMAGE=$LOCAL_GIT_BASE_IMAGE \
+      tests/local-git-server
+
+.prepare-tests: &prepare-tests
+  - python -m venv .venv
+  - pip install -e '.[dev]'
+  - source .venv/bin/activate
+  - cd tests
+
+.prepare-t4c-server-tests: &prepare-t4c-server-tests
+  - export DOCKER_PREFIX=$T4C_SERVER_REGISTRY/
+  - export T4C_SERVER_TAG=$T4C_SERVER_TAG
+  - export LOCAL_GIT_TAG=latest
+  - export DOCKER_NETWORK=bridge
+  - *local-git-server
+  - *prepare-tests
+  - git clone $T4C_SERVER_TEST_DATA_REPO
```

Comment:
```zsh
docker inspect -f "{{json .NetworkSettings.Networks }}" $(docker ps -q -f "label=com.gitlab.gitlab-runner.job.id=$CI_JOB_ID" -f "label=com.gitlab.gitlab-runner.type=build") | jq 'keys[0]' | head -n 1
```

With this command, you can identify the current container ID, then inspect the container, get the network the container is in as json and take the first "key". We have the bridge network in our environment, in another environments, it may be different.